tests/requirements-py3.txt

```diff
@@ -17,5 +17,5 @@ uvloop; platform_system != "Windows"
 bpython
 brotlipy  # optional for HTTP compress downloader middleware tests
 zstandard  # optional for HTTP compress downloader middleware tests
-ipython
```

Comment:
Not specifying a version leads to [failed build](https://github.com/scrapy/scrapy/pull/4922/checks?check_run_id=1531501862), specifying a minimum version leads to the job hanging indefinitely.